Projects/MDBMemberQuiz/app/src/main/java/com/example/radhika/mdbmemberquiz/QuizActivity.java

```diff
@@ -0,0 +1,223 @@
+                                score = 0;
+                                Intent i = new Intent(getApplicationContext(), MainActivity.class);
+                                startActivity(i);
+                            }
+                        })
+                        .setNegativeButton("Cancel", new DialogInterface.OnClickListener() {
+                            @Override
+                            public void onClick(DialogInterface dialog, int which) {
+                                nextMember();
+                            }
+                        }).show();
+                break;
+            case R.id.memberPicture:
+                timer.cancel();
+                Intent intent = new Intent(Intent.ACTION_INSERT);
+                intent.setType(ContactsContract.Contacts.CONTENT_TYPE);
+                intent.putExtra(ContactsContract.Intents.Insert.NAME, currMember);
+                startActivity(intent);
+                break;
+        }
+
+    }
+    // get an array with 4 member names that will act as multiple choice options for user
+    private String[] getOptions() {
+        String[] options = new String[4];
+        for (int i = 0; i < options.length; i++) {
+            while (options[i] == null) {
+                int rand = (int) (Math.random() * members.length);
+                if (!contains(members[rand], options)) {
+                    options[i] = members[rand];
+                }
+            }
+        }
+        return options;
+    }
+
+    // checks if String x is in array options
+    private boolean contains(String x, String[] options) {
+        for (int i = 0; i < options.length; i++) {
+            if (options[i] != null && options[i].equals(x)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    /**
+     * ATTENTION: This was auto-generated to implement the App Indexing API.
+     * See https://g.co/AppIndexing/AndroidStudio for more information.
+     */
+    public Action getIndexApiAction() {
+        Thing object = new Thing.Builder()
+                .setName("Quiz Page") // TODO: Define a title for the content shown.
+                // TODO: Make sure this auto-generated URL is correct.
+                .setUrl(Uri.parse("http://[ENTER-YOUR-URL-HERE]"))
+                .build();
+        return new Action.Builder(Action.TYPE_VIEW)
+                .setObject(object)
+                .setActionStatus(Action.STATUS_TYPE_COMPLETED)
+                .build();
+    }
+
+    @Override
+    public void onPause() {
+        super.onPause();
+        timer.cancel();
+    }
+    @Override
+    public void onStart() {
+        super.onStart();
+        // ATTENTION: This was auto-generated to implement the App Indexing API.
+        // See https://g.co/AppIndexing/AndroidStudio for more information.
+        client.connect();
+        AppIndex.AppIndexApi.start(client, getIndexApiAction());
+    }
+    @Override
+    public void onRestart() {
```

Comment:
typo - yes
timer - calls nextMember which starts timer again with a new member (didn't want users to cheat by leaving the app to think and coming back because I tried to do that lol)